org-code-javabuilder/validation/src/main/java/org/code/validation/EventTypeMapper.java

```diff
@@ -0,0 +1,34 @@
+package org.code.validation;
+
+import org.code.neighborhood.NeighborhoodSignalKey;
+
+public class EventTypeMapper {
+  // Map NeighborhoodSignalKey to EventType. This allows us to change NeighborhoodSignalKeys without
+  // needing to update the validation api.
```

Comment:
I'm a bit on the fence about whether it makes sense to just use the NeighborhoodSignalKey enum directly - it seems like these need to be tightly coupled anyway since this is testing neighborhood code. I'm curious if you can share more detail about why you decided to do it this way. 

Edit: now that I'm thinking through this, I'm realizing it's likely because EventType is user-facing (i.e. where the user is the levelbuilder) and NeighborhoodSignalKey is an internal implementation detail of Javabuilder. So we may change these independent of each other and we definitely don't want to have to go through and edit all the level sources if we change one of the NeighborhoodSignalKey .

Reply:
yep that was why. Also because EventType is a clearer name for levelbuilders to use instead of having them use NeighborhoodSignalKey.